integration/test_web.py

```diff
@@ -0,0 +1,155 @@
+import sys
+import time
+import shutil
+import json
+import urllib2
+from os import mkdir, unlink, utime
+from os.path import join, exists, getmtime
+
+import util
+
+import requests
+import pytest_twisted
+
+
+def test_index(alice):
+    """
+    we can download the index file
+    """
+    util.web_get(alice._node_dir, "")
+    # ...and json mode is valid json
+    json.loads(util.web_get(alice._node_dir, "?t=json"))
+
+
+def test_upload_download(alice):
+    """
+    upload a file, then download it via readcap
+    """
+
+    FILE_CONTENTS = "some contents"
```

Comment:
Bytes or unicode?  Same question for all other string literals in this PR, I suppose.